cmd/aws-lambda-rie/http.go

```diff
@@ -16,15 +21,85 @@ func startHTTPServer(ipport string, sandbox *rapidcore.SandboxBuilder, bs intero
 		Addr: ipport,
 	}
 
+	log.Warnf("Listening on %s", ipport)
+
+	maxInvocations := -1 // -1 means unlimited invocations
+	// Get max invocations from environment variable
+	maxInvocationsStr := os.Getenv("AWS_LAMBDA_SERVER_MAX_INVOCATIONS")
+	if maxInvocationsStr != "" {
+		if maxInvocationsInt, err := strconv.Atoi(maxInvocationsStr); err == nil {
+			maxInvocations = maxInvocationsInt
+		} else {
+			log.Panicf("Invalid value for AWS_LAMBDA_SERVER_MAX_INVOCATIONS: %s", maxInvocationsStr)
+		}
+	}
```

Comment:
Read the `AWS_LAMBDA_SERVER_MAX_INVOCATIONS` environment variable into the `maxInvocations` variable.